hw/top_earlgrey/data/ip/chip_aes_testplan.hjson

```diff
@@ -111,7 +111,8 @@
       stage: V2
       si_stage: SV3
       lc_states: ["PROD"]
-      tests: []
+      tests: ["chip_sw_aes_interrupt_encryption"]
```

Comment:
This field is used for to link dv tests listed in the file `hw/top_earlgrey/dv/chip_sim_cfg.hjson`.
So in this case does not apply, can you please drop this change?

Reply:
Sure.  Just updated the file